src/scripts/main.js

```diff
@@ -1,7 +1,107 @@
 'use strict';
 
-// Uncomment the next lines to use your game instance in the browser
-// const Game = require('../modules/Game.class');
-// const game = new Game();
+const Game = require('../modules/Game.class');
+const game = new Game();
+const gameScore = document.querySelector('.game-score');
+const startButton = document.querySelector('.start');
+const messageWin = document.querySelector('.message-win');
+const messageLose = document.querySelector('.message-lose');
+const messageStart = document.querySelector('.message-start');
 
-// Write your code here
+const fields = document.querySelectorAll('.field-cell');
+
+document.addEventListener('keydown', keypressHandler);
+startButton.addEventListener('click', startGame);
+
+function keypressHandler(e) {
+  if (e.key === 'Escape') {
+    restartGame();
+  }
+
+  if (e.key === 'Enter') {
+    startGame();
+  }
+
+  if (game.getStatus() === 'playing') {
```

Comment:
Ensure that the game status is checked before processing any moves. This is already done, but make sure the logic is consistent and clear across all handlers.